src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -91,6 +91,54 @@ public static String composeClassHeaderCredentialsSampleCode(
     return writeSampleCode(Arrays.asList(initSettingsVarExpr, initClientVarExpr));
   }
 
+  public static String composeClassHeaderEndpointSampleCode(
```

Comment:
Could we please add a TODO to add unit tests for this method in the future?

Reply:
Added.